src/test/java/com/microsoft/sqlserver/jdbc/connection/RequestBoundaryMethodsTest.java

```diff
@@ -412,4 +436,56 @@ private void compareValuesAgainstConnection(SQLServerConnection con, boolean aut
     private void generateWarning(SQLServerConnection con) throws SQLException {
         con.setClientInfo("name", "value");
     }
+
+    private boolean isVerified(Method method) {
+        return (!Modifier.isPublic(method.getModifiers()) || Modifier.isStatic(method.getModifiers())
+                || method.getName().startsWith("get") || getVerifiedMethodNames().contains(method.getName()));

```

Comment:
why do we allow any method that starts with "get" to be verified? if I added a new connection property and added a public getter for it, isn't it something that the boundary request method needs to handle? (it would likely get caught by the setter that comes with it, though)

Reply:
The point of beginRequest() is to "memorize" the connection state and its properties. Getting a connection property doesn't affect the properties so it shouldn't matter. On the other hand, it is kind of hacky and I wouldn't be opposed to triggering this error on all newly added public methods, just to be safe.

Reply:
Well, a getter won't change the connection local state. And I didn't want to hard-code all of them in the test.